recipes/_freebsd.rb

```diff
@@ -21,4 +21,7 @@
   package 'devel/gmake'
   package 'devel/autoconf'
   package 'devel/m4'
+  package 'devel/gettext'
+  # Only install gcc on freebsd 9.x - 10 uses clang
+  package 'lang/gcc49' if node['os_version'].to_i <= 903_000
```

Comment:
node['platform_version'].to_i will result in 9, which is a bit more obvious when reading the code